tests/system/large/functions/test_remote_function.py

```diff
@@ -1359,6 +1360,51 @@ def square_num(x):
         )
 
 
+@pytest.mark.flaky(retries=2, delay=120)
+def test_remote_function_warns_default_cloud_function_service_account(scalars_dfs):
+    project = "bigframes-dev-perf"
+
+    gcf_service_account = "default"
+
```

Comment:
If they explicitly provide `"default"` it should _not_ warn. Only if they don't provide anything.

```suggestion
```